packages/react-i18n/src/constants/index.ts

```diff
@@ -3,9 +3,17 @@ export enum DateStyle {
   Short = 'Short',
   Humanize = 'Humanize',
   Time = 'Time',
+  DateTime = 'DateTime',
 }
 
 export const dateStyle = {
+  [DateStyle.DateTime]: {
```

Comment:
Silly nitpick: could we keep he ordering of the keys in this const and in DateStyle the same?

Reply:
Yes we should, updated the code to reflect this 👍